pages/admin/login/index.tsx

```diff
@@ -47,20 +74,45 @@ const Login: NextPage = () => {
           alt="UW Blueprint Logo"
           style={{ position: "relative", top: 37, left: 26 }}
         />
-        <form className="flex flex-col space-y-[26px] pl-[88px] mt-[90px]">
+        <form className="flex flex-col pl-[88px] mt-[90px]">
+          {incorrectCredentialsError && (
+            <div className="bg-error text-white p-[5px] text-xs rounded-md mb-[10px] flex items-center justify-center gap-[3px]">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="currentColor"
+                viewBox="0 0 24 24"
+                stroke-width="1.5"
+                stroke="#EF5D5D"
+                className="w-6 h-6"
+              >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  d="M12 9v3.75m9-.75a9 9 0 11-18 0 9 9 0 0118 0zm-9 3.75h.008v.008H12v-.008z"
+                />
+              </svg>
```

Comment:
Would it be possible to put this icon into its own component, so that we can reuse it elsewhere?

Reply:
can you follow up on this?  lmk if you think it's better as-is, or want to move it to another pr later